.gitignore

```diff
@@ -6,13 +6,11 @@
 # Terraform specific files
 terraform.tfplan
 terraform.tfstate
-*.terraform.lock.hcl
-./*.tfstate
```

Comment:
`terraform.tfstate` is the default name. the added `*.terraform*` rule would not capture this

Reply:
There's a line for `terraform.tfstate` on line 8. Is it possible that the state file could have a different name? 

My thought was that there was already a rule for it so the redundant rule was unnecessary.